reference/5.1/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -359,14 +359,16 @@ You can pipe any object to `Out-File`.
 
 ## NOTES
 
-If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a formatting cmdlet
-before rendering it.
+Input objects are automatically formatted as there would be in the terminal, but you can use a
+`Format-*` cmdlet to explicitly control the formatting of the output to the file. For example,
+`Get-Date | Format-List | Out-File out.txt`
 
-To send a PowerShell command's output to the `Out-File` cmdlet, use the pipeline. You can store data
-in a variable and use the **InputObject** parameter to pass data to the `Out-File` cmdlet.
+To send a PowerShell command's output to the `Out-File` cmdlet, use the pipeline. Alternatively, you
+can store data in a variable and use the **InputObject** parameter to pass data to the `Out-File`
+cmdlet.
 
-`Out-File` sends data but it does not produce any output objects to the pipeline. It only writes
-data to a file.
+`Out-File` saves data to a file but it does not produce any output objects to the pipeline. It only
+writes data to a file.
```

Comment:
Given the reformulated first sentence in this paragraph, I think we can simply _remove_ "It only
data to a file.	writes data to a file." now.

Reply:
🤦 you are right, fixed